retention/options.go

```diff
@@ -37,16 +38,22 @@ const (
 	// defaultBufferPast is the default buffer past limit
 	defaultBufferPast = 10 * time.Minute
 
-	// defaultBufferDrain is the default buffer drain
-	defaultBufferDrain = 2 * time.Minute
```

Comment:
Yep. Because we don't need a separate duration per namespace, moved this to `storage.Options`, and renamed to `tickInterval`.